.travis.yml

```diff
@@ -1,12 +1,25 @@
 language: python
-python: 
-  - 2.7
-  - 3.6
-env:
-  - TOX_ENV=py27
-  - TOX_ENV=py36
-  - TOX_ENV=pep8
+
+
+branches:
```

Comment:
I don't think we want this for branches, as we'll need it for master once this has merged (and stable branches once backported).

Reply:
Sure, I will remove "branches" attribute.

Reply:
Updated in latest commit